multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/model/SupportedParameters.java

```diff
@@ -125,8 +125,10 @@
     public static final String DEFAULT_CONTAINER_NAME = "default-container-name";
     public static final String DEFAULT_XS_APP_NAME = "default-xsappname";
     public static final String TYPE = "type";
-    public static final String IGNORE_UPDATE_ERRORS = "ignore-update-errors";
-
+    public static final String SKIP_PLAN_UPDATE = "skip-plan-update";
+    public static final String SKIP_CREDENTIALS_UPDATE = "skip-credentials-update";
```

Comment:
Rename to `skip-parameters-update`.

Also, since this is a UX change, we should have a short 15-30 min meeting with the rest of the team, so that everyone has the chance to give feedback, share his concerns, etc. One thing to discuss is whether this parameter should be called `skip-parameters-update` or `skip-service-instance-parameters-update`, since the first might be a bit ambiguous, as it's a bit unclear whether it concerns resource or service instance parameters.

Reply:
Changed to a map called `skip-service-updates` with 3 boolean fields: `parameters`, `tags`, `plan` as per the team's decision.